Taskfile.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+env:
+  IMAGE_NAME: ac-ui
+  TAG: latest
+
+tasks:
+  install:
+    desc: Install dependencies
+    sources:
+      - package.json
+    cmds:
+      - npm install
+    internal: True
+
+  build:
+    desc: Build docker image
+    deps: ["package.json"]
```

Comment:
i think the dep name should ne the name of the task i.e. "install" in this instance